### DISCUSSION TITLE
Can't seem to get SVG files to upload.

### DISCUSSION THREAD

**Discussion by ahmetskilinc** [payloadcms/payload#666] [Q&A]
When ever I try to upload an SVG file I get the error `ValidationError: The following fields are invalid: sizes.large.mimeType`

My upload/media collection currently looks like this: 

```javascript
const Media = {
    slug: "media",
    access: {
        read: () => {
            return true;
        },
    },
    upload: {
        staticURL: "/src/media",
        staticDir: "src/media",
        imageSizes: [
            {
                name: "large",
                width: 1024,
                height: null,
                crop: "centre",
            },
        ],
        adminThumbnail: "thumbnail",
        mimeTypes: ["image/*", "image/svg+xml"],
    },
};

export default Media;
```

Is there something I've missed. I've searched all around and can't seem to find anything.

**Comment by denolfe** [comment]
Hey @ahmetskilinc, there does appear to be an issue with specifying the `mimeTypes` property here. I was able to recreate. I'll convert this to an issue.

**Reply by slawekniewiadomski** [reply]
I think i've found something maybe related, don't have time now to take a second look, but maybe problem stems from inconsistent letter 't' case in word mimetype a.k.a mime**T**ype somewhere? for example:
https://github.com/payloadcms/payload/blob/bf48fdf18961a2e57bcc5aae73de4c569e97e42b/src/uploads/uploadFile.ts#L66-L86
This inconsistency can be found in other files, maybe some wild undefined or null fails silently here.
Also, line 75 seems to exclude svg?

**Reply by ahmetskilinc** [reply]
Thanks @denolfe and @slawekniewiadomski line 75 does exclude svg so I may be getting an error because of that. I have tried with the same images converted to png and I get an error still (same error message) but this is because of size I'm assuming. The width on the largest images are 948px and smallest is 450px, don't know if that will help.

**Comment by DanRibbens** [comment]
The issues for mimetypes and filesizing should be addressed for everyone now. Please upgrade payload to `0.18.5` and give it a shot again.

You *may* need to remove the unique index for your collection filesizes.[name].filename. This was 1/2 of the bug we were seeing and it won't be removed without manually dropping the collection or just the indexes for your sizes manually.

**Reply by ahmetskilinc** [reply]
Thanks so much! this has worked!
